PyPDF2/filters.py

```diff
@@ -61,7 +61,7 @@ def decompress(data: bytes) -> bytes:
     except zlib.error:
         d = zlib.decompressobj(zlib.MAX_WBITS | 32)
         result_str = b""
-        for b in [data[i : i + 1] for i in range(len(data))]:
```

Comment:
I think those are added by `black` (E203) and will be necessary to be added again. I really want the black formatter to be applied (although I don't want to force contributors to do it; the less friction for a new contribution the better - I can apply black later as well)